lib/annotate_rb/runner.rb

```diff
@@ -23,23 +23,18 @@ def running?
     end
 
     def run(args)
-      config_file_options = ConfigLoader.load_config
       parser = Parser.new(args, {})
-
       parsed_options = parser.parse
       remaining_args = parser.remaining_args
 
-      options = config_file_options.merge(parsed_options)
+      AnnotateRb::RakeBootstrapper.call
 
-      @options = Options.from(options, {working_args: remaining_args})
-      AnnotateRb::RakeBootstrapper.call(@options)
+      options = ConfigLoader.load_config.merge(parsed_options)
```

Comment:
fyi `options` is not used here as complained by linter. 

Can run `bundle exec standardrb -A` to lint the rest